packages/field-base/src/clear-button-mixin.js

```diff
@@ -103,7 +103,7 @@ export const ClearButtonMixin = (superclass) =>
      * @protected
      */
     _onClearAction() {
-      this.clear();
+      this._inputElementValue = '';
```

Comment:
It's enough to only update the input element's value. Everything else will be updated on the input event fired below.